geometry/project/src/main/java/com/esri/samples/project/ProjectSample.java

```diff
@@ -58,9 +60,15 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create a map with a web mercator basemap
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with a web mercator spatial reference and a basemap style
       ArcGISMap map = new ArcGISMap(SpatialReference.create(3857));
```

Comment:
Same here, we could write `SpatialReferences.getWebMercator()`, and possibly even leave it out?
For the comment, maybe we should capitalise `Web Mercator` as it's an established term? Not 100% sure though
```suggestion
      // create a map with the Web Mercator spatial reference and a basemap style
      ArcGISMap map = new ArcGISMap(SpatialReferences.getWebMercator());
```

Reply:
I've tidied this up a bit